www/creating-zooming-images.html

```diff
@@ -191,6 +196,31 @@ <h2>Creating Zooming Images</h2>
         <td>Free desktop app for locally viewing DZI files for Windows. Linux soon.</td>
         <td><a href="/examples/tilesource-dzi/">DZI</a></td>
     </tr>
+    
+    <tr>
+        <td colspan="3"><h4>Single-file Tile formats</h4> - see complete <a href="https://github.com/openseadragon/openseadragon/issues/944">discussion thread</a>. Useful for faster/easier file operations and maintenance on S3/file servers FS.</td>
```

Comment:
Thank you for adjusting the h4. Now the `- see complete` looks odd on the next line. It should be the start of a sentence, like `See complete`.